doc/sample_cookieblocklist.txt

```diff
@@ -148,6 +149,8 @@ fastly.net
 fbcdn.net
 attachment.fbsbx.com
 feedburner.com
+api.filepicker.io
+dialog.filepicker.io
```

Comment:
looks like filepicker aka filestack uses www.filepicker.io - can you add that to the yellowlist?

Reply:
Thanks! Adding all of `filepicker.io` in #1677.